src/js/index.js

```diff
@@ -35,7 +35,7 @@ store.subscribe(throttle(model.update, 16));
 var ui = require('./components');
 
 // name model Scene
-model.Scene.name = 'Scene';
+model.Scene._name = 'Scene';
```

Comment:
My question is, why was that change made. We have a considerable amount of code which assumes that the mark primitives (of which scene is one, even though it is handled a little differently) behave the same way in certain fashions, one of which is that they all can consistently be relied upon to have an `_id`, `type` and `name`. If we're changing to `_name` all over, I have a lot of work I need to do; and if we're doing it as a one-off for scene I want to understand why, because that has implications for the existing reducer code I've written. The motivation of the change is what I'm looking for.